CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - SecureNN with int64 support has landed, a tensorflow build with int64 matmul support must be used. See details on how to do that [here](./README.md#securenn-int64-support)
 - Cryptographically secure random numbers feature has been implemented.
-- Three new ops are now supported: Pad, BatchToSpaceND, SpaceToBatchND
+- Seeds are sent from crypto-producer instead of full random tensors during masking and multiplication related ops (e.g. mul, matmul, conv2d, square).
```

Comment:
Perhaps reword a bit to say:

- "We now send seeds from the crypto-producer instead of full tensors when masking. This reduces the amount of data sent across the wire between parties during multiplication effecting layers such as mul, matmul, conv2d, and square."

Just so it's clear that this reduces data across wire. Feel free to use mine or modify yours as you see fit, just wanted to demonstrate what I meant.

Reply:
Borrowed some of it! Just wanted to make it clear that seeds are sent during masking but also during multiplication as well. See updated text!